### PR TITLE
Add a rake task for sync'ing rules + policies, and add some convenience methods for creating human-readable rules

### LINKED ISSUE #3183: Create a process for keeping cached FOLIO policies in sync

### PR DESCRIPTION
**Pull request by cbeer**
Fixes #3183

### CODE REVIEW

File: config/settings.yml [hunk 1]
```diff
@@ -78,6 +78,9 @@ LIB_GUIDES:
   NUM_RESULTS: 3
   SITE_ID: 12345
 
+folio:
+  url: <%= ENV['OKAPI_URL'] || 'http://okap-test.stanford.edu' %>
```
Comment (thatbudakguy) [inline config/settings.yml:82]: okapi-test?

File: app/services/folio/types.rb [hunk 2]
```diff
@@ -0,0 +1,79 @@
+module Folio
+  class Types
+    class << self
+      delegate :policies, :circulation_rules, :criteria, to: :instance
+    end
+
+    def self.instance
+      @instance ||= new
+    end
```
Comment (thatbudakguy) [inline app/services/folio/types.rb:9]: is this just the singleton pattern? I thought you could do `include Singleton` or something

File: app/services/folio/circulation_rules/rule.rb [hunk 3]
```diff
@@ -6,10 +6,48 @@ module CirculationRules
...
+                        else
+                          type_map.call(v)
+                        end
+          "#{k}: #{debug_value}"
+        end.join(" & ")
+      end
+
+      def to_policy_debug_s
+        policy.map { |k, v| "#{k}: #{Folio::Types.policies.dig(k.to_sym, v, 'name')&.strip || v}" }.join(", ")
+      end
```
Comment (thatbudakguy) [inline app/services/folio/circulation_rules/rule.rb:50]: a docstring for any or all of these methods that illustrates what the output looks like would be really nice

**Review by thatbudakguy — APPROVED** [review]
overall this looks great; no hard blockers to merging

File: config/folio/lost_item_feeds_policies.json
[the remark this answers is not in the document]
Comment (cbeer) [reply config/folio/lost_item_feeds_policies.json:1]: Yep, I must have typoed it and forgot to clean out the directory.

File: app/services/folio/types.rb (hunk 2 above)
Comment (cbeer) [reply app/services/folio/types.rb:9]: You could, except if you want to, say, test it with different parameters.

**Pull request merged**